### PR TITLE
Document changelog

### PR DESCRIPTION
**Pull request by vahidbaharvand**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+This file includes the major changes to the Test4z sample project.
+## [2.0.0] 2022-03-22
+### Added
+- Added z/OS Connect supportive case.
+- Added CA View supportive case.
+- Added the mock server.
```
Comment (vibhor-bc) [inline CHANGELOG.md:6]: We also have samples demonstrating compare with different layouts & restructuring of samples.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+This file includes the major changes to the Test4z sample project.
+## [2.0.0] 2022-03-22
```
Comment (vibhor-bc) [inline CHANGELOG.md:2]: The date should be changed.

File: README.md [hunk 3]
```diff
@@ -44,8 +44,8 @@ Once you have met the prerequisites,
 
 ## Notes
  
-1. Exclude feature of the Compare endpoint requires a certain File Master Plus version 
-   12.0 and the required PTF installation. Make sure you have the required PTF to be able to use the field exclusion.
-  
+- Exclude feature of the Compare endpoint requires a certain File Master Plus version 12.0 and the required PTF installation. Make sure you have the required PTF to be able to use the field exclusion.
```
Comment (vibhor-bc) [inline README.md:47]: I think we should not mention the backend service components. It should be taken care by the service installation admin. The version of samples are compatible with the specific version of service installation.

**Review by vibhor-bc — CHANGES_REQUESTED** [review]

**Review by vibhor-bc — APPROVED** [review]

**Review by hizir-bc — APPROVED** [review]

**Review by hizir-bc — CHANGES_REQUESTED** [review]
For this release there should be an item for **Restructuring the samples**

**Review by hizir-bc — APPROVED** [review]

**Pull request merged**